### PR TITLE
handle errors in apps while registering commands

### PR DESCRIPTION
**Pull request by icewind1991**
Don't completely break the console just because a single app fucked up

### CODE REVIEW

**Comment by nickvergessen** [comment]
👍

File: lib/private/Console/Application.php [hunk 1]
```diff
@@ -45,18 +46,21 @@ class Application {
...
+	/** @var ILogger  */
+	private $logger;
 
 	/**
 	 * @param IConfig $config
 	 * @param EventDispatcherInterface $dispatcher
 	 * @param IRequest $request
 	 */
-	public function __construct(IConfig $config, EventDispatcherInterface $dispatcher, IRequest $request) {
+	public function __construct(IConfig $config, EventDispatcherInterface $dispatcher, IRequest $request, ILogger $logger) {
```
Comment (LukasReschke) [inline lib/private/Console/Application.php]: Missing PHPDoc update

Comment (icewind1991) [inline lib/private/Console/Application.php]: done

**Review by LukasReschke — APPROVED** [review]

**Comment by LukasReschke** [comment]
LGTM

**Pull request merged**